config/locales/ar.yml

```diff
@@ -102,7 +102,7 @@ ar:
       login:
         title: "دخول"
         remember_me: "تذكرني"
-        submit: "تسحيل"
+        submit: "تسجيل"
```

Comment:
When I translated the original and the new version for this change, both return "Sign Up" as the English translation, yet its for the **login** locale.

I noticed the correction you made is the same as the second word in the `sign_up.submit` locale. Translating that full value, "تسجيل الدخول" actually returns "Sign In". 😳

Reply:
@javierjulio I appreciate your taking the time to check this.

Regarding the original word "تسحيل". It's not an actual word in Arabic that's why Google translate shows results for the corrected word instead as indicated by the note on the left saying "Showing translation for تسجيل".

As for the issue you spotted. You're right. I was focused on fixing the typo and completely missed noticing it.
It seems the locales for "sign up" and "sign in" have been misplaced and nobody noticed before since Admin users don't sign up. Also using 'تسجيل' (record / register) in conjunction with the title "دخول" (login / entry) would implicitly mean "تسجيل الدخول" (i.e. to sign in), just more concise.

So, to correct this I suggest we use:
- `sign_up.title`: "تسجيل" and `sign_up.submit`: "تسجيل"
- `login.title`: "تسجيل الدخول" and `login.submit`: "تسجيل الدخول"

I'll update the PR to reflect that if it's fine with you.

Reply:
@mshalaby you're very welcome. I love seeing these locale contributions come in. We really appreciate you and others contributing them!

You don't have too but the suggested change looks great to me as long as you are ok with updating, let's do it. We can get that change in. Thank you!

Reply:
Thanks @javierjulio!

I've pushed the fixes we mentioned.

While doing so I noticed another unrelated issue.
The word order is reversed for `new_model` translation:
https://github.com/activeadmin/activeadmin/blob/7b45fbf429542ebd5ddb137a499cedc21ce20773/config/locales/ar.yml#L11
The correct order should be:
```yml
new_model: "%{model} جديد"
```
Should I create a separate issue for that? or just push another commit to fix it here?

Reply:
You're welcome! Up to you, either is fine for me since it's a small change. Thank you! ❤️

Reply:
Pushed the fix. You're very welcome @javierjulio :pray: